src/http/get-file.js

```diff
@@ -3,9 +3,8 @@ import compressFile from './compress-file';
 
 export default opts => {
   const { storage, fileKey, req, res, contentType, auth } = opts;
-
   auth(err => {
-    const acl = err ? 'public' : 'private'; // if auth fails, pass as public request
+    const acl = err && !auth.skip ? 'public' : 'private'; // if auth fails, pass as public request
```

Comment:
`err || auth.skip === true`

Reply:
If `auth.skip === true`, we wanna ignore `auth err` and create knox client to get a photo

Reply:
Which my code will do.